Makefile

```diff
@@ -88,3 +88,7 @@ deploy_release:
 
 release: tag_release deploy_release
 
+# Make sure the following matches what is in the .travis.yml file.
+flake:
+	flake8 . --exclude=./docs/conf.py --ignore=E123,E133
+
```

Comment:
according to the docs we could make a setup.cfg in the root of zamboni and then "flake8" from the command line in the root would work

http://flake8.readthedocs.org/en/latest/config.html#default